src/Infrastructure/Adapter/APIAdresseGeocoder.php

```diff
@@ -101,44 +101,46 @@ public function computeCoordinates(string $address): Coordinates
         return Coordinates::fromLonLat($lonLat[0], $lonLat[1]);
     }
 
-    public function findAddresses(string $type, string $search): array
+    public function findAddresses(string $search): array
     {
         $response = $this->apiAdresseClient->request('GET', '/search/', [
             'headers' => [
                 'Accept' => 'application/json',
             ],
             'query' => [
                 'q' => $search,
-                'type' => $type,
                 'autocomplete' => '1',
                 'limit' => 7,
             ],
         ]);
 
         try {
             $data = $response->toArray(throw: true);
-        } catch (\Exception $exc) {
-            \Sentry\captureException($exc);
+            $addresses = [];
 
-            return [];
-        }
+            foreach ($data['features'] as $feature) {
+                $type = $feature['properties']['type'];
+
+                if ($type === 'housenumber') {
+                    continue;
+                }
 
-        $addresses = [];
+                $label = match ($type) {
+                    'street', 'locality' => $feature['properties']['label'],
```

Comment:
@aureliebaton suggérait d'ajouter une virgule pour la lisibilité

> Pour la lisibilité, est-ce qu'on peut avoir une virgule après la rue, comme dans Maps et autres

https://github.com/MTES-MCT/dialog/issues/213#issuecomment-1495908873

```suggestion
                    'street', 'locality' => sprintf('%s, %s %s', $feature['properties']['name'], $feature['properties']['postcode'], $feature['properties']['city']),
```

Reply:
Tant qu'on n'a aucune idée de comment les GPS traiteront la donnée, je suis d'accord pour formater de cette façon.

Reply:
Je pense qu'on finira par stocker une version parsée, avec roadName/city/postCode

Car dans l'UI on souhaite aussi afficher les infos sans forcément utiliser la valeur de `address` directement, on dirait

> * Rue Hyacynthe Ménager (76300)
> * Sotteville-lès-Rouen

Reply:
Pas forcément besoin de stocker dans différents champs, on peut également le formater à l'affichage. Encore une fois, il faudra voir à l'usage et notamment comment les GPS utiliseront cette data.